modules/internal/ChapelArray.chpl

```diff
@@ -3697,7 +3697,7 @@ module ChapelArray {
   proc =(ref a: _distribution, b: _distribution) {
     if a._value == nil {
       __primitive("move", a, chpl__autoCopy(b.clone(), definedConst=false));
-    } else if a._value._doms.size == 0 {
+    } else if a._value._doms_size == 0 {
```

Comment:
Not to defend that identifier but, can we change this to `_doms_containing_dist` so that our terminology is more consistent?

Reply:
Yeah, changed